### ISSUE TITLE
There are over 355 OPEN feature requests that span more than four years. 

### ISSUE THREAD

**Issue by anoduck** [orgzly/orgzly-android#1009]
I know this might come off as rude, but it does need to be addressed. There are over 355 OPEN feature requests that span more than four years, many of which have not been responded, and it appears there is no plans to address and process them. These feature requests span from features already available, features quite easy to implement, to features utterly absurd and pointless.

Somehow these features need to be processed and addressed, because ignoring them is just filling up the issues queue, and eventually will make navigating the queue difficult. Furthermore, not processing the issues make developers appear to not be interested or concerned with creative and sometimes beneficial contributions made by users. Leaving all feature requests unanswered makes it appear that all suggestions are also equivocally superfluous, which obviously they are not.

I would like to contribute to the development of this application as much that I can, but seeing how I do not know any kotlin, nor have any need for it outside this application, I have been hesitant. Perhaps, I could be of some service handling these unattended to feature requests and provide help writing and formatting documentation. My collegiate studies were all in philosophy, theology, ethics, and business. So I can write.

**Comment by mtekman** [comment]
Developers aren't beholden to their fans and their various whims; their time and motivations is theirs and theirs alone, as it should be. Developing a free and open source app should be **FUN** and not a chore, rewarded by the appreciation of its users.

It does indeed suck that neither you nor I as capable software devs in a variety of languages know how to meaningfully contribute to this project¹²³, but that is not the fault of the developer.

As you say though, one thing we can do to contribute is create better documentation. 

Possible entry point here:
https://github.com/orgzly/orgzly-android/issues/24#issuecomment-1335137275

*******

_Footnotes / Rants:_

¹ and I see this as a failure of App development languages in general...
² due to the constant effort of having to keep up with the shifting software paradigms every year or so...
³ to the point that whatever you learned in app development 3 years ago is already redundant information...

**Comment by anoduck** [comment]
@mtekman Your footnotes do not link to anything, so their context is somewhat moot at this moment. 

I appreciate you sharing your thoughts and opinions with me, I don't agree with some of the viewpoints, but appreciate the sharing nonetheless. Although, they don't directly address the issue at hand.

**Comment by komali2** [comment]
Philosophy eh? 

Take the bigger picture then.

You saw a thing done not how you want it done. That thing is an android app for displaying a relatively niche structured plain text file format that predates markdown, or: a todo app. 

You see a lot of open feature requests and think, "huh, people should work on these!" You spent some amount of time to come here and tell us.

What have you accomplished?

Consider: do you think others disagree? Do you think others weren't aware that having a ton of open feature requests isn't great? Do you think the missing piece between feature requests and implementation was a philosophy major making an issue to complain about it? 

Do you think this was an effective use of your time and energy, this specific thing about this specific niche app?

Consider: the app works fine. It gets the job done. The organization of the github repo for it isn't correlated with the functioning of the app. Stuff still gets done on it. Not perfect! But good enough. 

> I would like to contribute to the development of this application as much that I can, but seeing how I do not know any kotlin,

Given that obviously the missing piece between features requested and features implemented is almost certainly not "someone complaining about it," why did you do that, instead of either learning kotlin, or, functioning as a project manager and organizing feature requests into your idea of their priority and implementability? 

You didn't link any features in your comment. There's no utility to this issue. You've done nothing to further what is ostensibly your goal: to organize the github. 

> These feature requests span from features already available

Which? Put the work in to list them, help the repo get organized. 

> features quite easy to implement,

Which? Spec them, combine duplicates, tag them, and make them as easy as possible to implement. 

> to features utterly absurd and pointless.

Which? Start the discussion so the community can decide. 

> Furthermore, not processing the issues make developers appear to not be interested or concerned with creative and sometimes beneficial contributions made by users. 

The apparent mood of the developers is irrelevant to the quality of the app. Also, you clearly aren't interested or concerned enough to do something about it, why blame others for the same thing lol? 

>  Leaving all feature requests unanswered makes it appear that all suggestions are also equivocally superfluous, which obviously they are not.

Is that obvious? How about listing some here with your suggested priority, so we can discuss? 

> I have been hesitant.

In your philosophy studies I'm guessing you read a bit of Krotopkin? What's your theory on how things get done? Given that we exist outside a capitalist system right now (nobody's getting paid), why do you think this app exists and has contributors? 

Maybe time to stop hesitating? Your github is pretty fleshed out and busy, you seem very capable. In fact it looks like you're writing Python, it's not like you don't know how to code at all, I doubt you'd need to straight up learn Kotlin to contribute code. 

# TLDR 

Ok so do something about it

**Comment by xeruf** [comment]
> Spec them, combine duplicates, tag them

This would need Triage permission on the repo.

I think the point to take from this is that it would be helpful to expand the maintainer rights to multiple people, as the Orgzly organization currently only has a single member.
Our only alternatives now without that are opening PRs, of which there are already 25, or forking, which splits the community.

**Comment by anoduck** [comment]
@komali2 I think @xeruf covered my response thoroughly.

**Comment by doak** [comment]
@anoduck, I guess you already stumbled over https://github.com/orgzly-revived/orgzly-android-revived, don't you?

**Comment by anoduck** [comment]
> @anoduck, I guess you already stumbled over https://github.com/orgzly-revived/orgzly-android-revived, don't you?

Yes, and I have nothing but good things to say about it.
